app/src/scene/scene.cpp

```diff
@@ -13,7 +14,8 @@
 #include <glm/gtc/matrix_transform.hpp>
 #include <glm/gtc/type_ptr.hpp>
 
-Scene::Scene::Scene()
+Scene::Scene::Scene(int width, int height)
+    : _camera{ new Camera(45.0, static_cast<double>(width) / static_cast<double>(height), 100.0) }
```

Comment:
we need use static_cast here?

Reply:
Yes, the result of the division will have integer type without it